bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/dto/LocationEx.java

```diff
@@ -26,5 +29,17 @@
 public interface LocationEx extends Location {
     Optional<String> getCountry();
 
-    Optional<String> getTimezone();
+    @Nullable
+    String getTimezone();
+
+    public default ZoneId getZoneId() {
+        String local = getTimezone();
+        if (local != null) {
+            try {
+                return ZoneId.of(local);
+            } catch (DateTimeException ignore) {
+            }
+        }
+        return ZoneId.systemDefault();
```

Comment:
Woudl it be possible to do an extra step of fallback using the `TimeZoneProvider` ? That way the openHAB configured timezone woudl be used instead of the system timezone.
(Also applies to [HomeCapability.java:53)